File: VB/RegistrationForm.Lesson4/DataModel/EmployeesModelHelper.vb
Imports DevExpress.Mvvm
Imports System
Imports System.Collections.Generic
Imports System.Linq

Namespace RegistrationForm.DataModel
    Public NotInheritable Class EmployeesModelHelper

        Private Sub New()
        End Sub

        Private Shared Entities As New EmployeesDBEntities()
        Public Shared Sub AddNewEmployee(ByVal firstName As String, ByVal lastName As String, ByVal email As String, ByVal password As String, ByVal birthday As Date, ByVal gender As Integer)
            Dim emp As Employee = Entities.Employees.Create()
            emp.FirstName = firstName
            emp.LastName = lastName
            emp.Email = email
            emp.Password = password
            emp.Birthday = birthday
            emp.Gender = gender

            Entities.Employees.Add(emp)
            Entities.SaveChanges()
            Messenger.Default.Send(New DBEmployeesChangedMessage())
        End Sub
        Public Shared Function GetEmployees() As List(Of Employee)
            Return Entities.Employees.ToList()
        End Function
    End Class
    Public Class DBEmployeesChangedMessage
    End Class
End Namespace

File: CS/RegistrationForm.Lesson4/DataModel/EmployeesModelHelper.cs
using DevExpress.Mvvm;
using System;
using System.Collections.Generic;
using System.Linq;

namespace RegistrationForm.DataModel {
    public static class EmployeesModelHelper {
        static EmployeesDBEntities Entities = new EmployeesDBEntities();
        public static void AddNewEmployee(string firstName, string lastName, string email, string password, DateTime birthday, int gender) {
            Employee emp = Entities.Employees.Create();
            emp.FirstName = firstName;
            emp.LastName = lastName;
            emp.Email = email;
            emp.Password = password;
            emp.Birthday = birthday;
            emp.Gender = gender;

            Entities.Employees.Add(emp);
            Entities.SaveChanges();
            Messenger.Default.Send(new DBEmployeesChangedMessage());
        }
        public static List<Employee> GetEmployees() {
            return Entities.Employees.ToList();
        }
    }
    public class DBEmployeesChangedMessage { }
}
